data/translations/nl.ts

```diff
@@ -0,0 +1,417 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="nl_NL">
+<context>
+    <name>CustomEditor</name>
+    <message>
+        <location filename="../../src/customeditor/customeditor.cpp" line="32"/>
+        <source>RazerGenie - Custom Editor</source>
+        <translation>RazerGenie - Aangepaste bewerker</translation>
+    </message>
+    <message>
+        <location filename="../../src/customeditor/customeditor.cpp" line="78"/>
+        <location filename="../../src/customeditor/customeditor.cpp" line="89"/>
+        <source>Unknown matrix dimensions</source>
+        <translation>Onbekende matrix-afmetingen</translation>
+    </message>
+    <message>
+        <location filename="../../src/customeditor/customeditor.cpp" line="78"/>
+        <location filename="../../src/customeditor/customeditor.cpp" line="89"/>
+        <source>Please open an issue in the RazerGenie repository. Device name: %1 - matrix dimens: %2 %3</source>
+        <translation>Open een issue in de RazerGenie-repo. Apparaatnaam: %1 - Matrix-afmetingen: %2 %3</translation>
+    </message>
+    <message>
+        <location filename="../../src/customeditor/customeditor.cpp" line="96"/>
+        <source>Device type not implemented!</source>
+        <translation>Deze apparaatsoort wordt nog niet ondersteund!</translation>
```

Comment:
```suggestion
        <translation>Dit type apparaat wordt nog niet ondersteund!</translation>
```

Reply:
That was a stupid mistake of mine. Thanks for correcting! 🙂